FirebaseAdmin/FirebaseAdmin.Tests/Messaging/MessageTest.cs

```diff
@@ -1650,6 +1685,42 @@ public void WebpushNotificationWithInvalidHttpsLinkUrl()
             Assert.Throws<ArgumentException>(() => message.CopyAndValidate());
         }
 
+        [Fact]
+        public void AnalyticsLabelInvalid()
+        {
+            var message = new Message()
+            {
+                Topic = "test-topic",
+                Notification = new Notification()
+                {
+                    Title = "title",
+                    Body = "body",
+                },
+                FcmOptions = new FcmOptions()
+                {
+                    AnalyticsLabel = "label!",
```

Comment:
Add additional test cases for the cases when the label is empty and longer than 50.